Suggested tutorial revisions (#3032)
apollographql/apollo-ios: docs/source/tutorial/tutorial-execute-first-query.mdx

## Create an ApolloClient

> 1. Create a new Swift file within `RocketReserver` called `Network.swift`.  Set the target to `RocketServer` and add `import Apollo` to the top of the file.

Reviewer: ```suggestion
1. Create a new Swift file within `RocketReserver` called `Network.swift`.  Set the target to `RocketReserver` and add `import Apollo` to the top of the file.
```

Build and run your application. The web host might take a few seconds to spin up your GraphQL server if nobody's been using it recently, but once it's up, you should see a response that resembles the following:

<img src="images/query_console_log_sample.png" class="screenshot" alt="Query console log output, beginning with 'Success! Result: GraphQLResult...'"/>

This means the request was correctly executed and you now have a list of launch sites 🚀🚀🚀.

> Go ahead and remove the code added to the `init()` method and replace it with a TODO for later:

Author: @calvincestari @BobaFetters I totally blew by this section in my run-through.  I'm happy to throw this change away if it's a little "too much" but maybe an explicit step to replace the fetch code with a TODO will call more attention to it?  I might just be imposing my user error on the tutorial but let me know if this is worth doing.

Reviewer: Do you think we should have the user replace the code with a `TODO` pointing to section 13, or have a `TODO` for section 13 already there in the starter project and have the user add this temporary code below it, and then instruct to remove it to end up back with the provided code snippet?

Reviewer 2: I think having a TODO already in the starter project, adding code below. The removal at the end should then be backed up by the still-existing TODO marker which is clear they still need make some change (remove the code).

> Go ahead and remove the code added to the `init()` method and replace it with a TODO for later:

Reviewer: ```suggestion
Go ahead and remove the code added to the `init()` method so there is just the TODO for later:
```
